client/src/backoffice/InboxListPage.tsx

```diff
@@ -76,12 +76,17 @@ const InboxListPage: FC = () => {
     }
 
     try {
-       const response = await getNotifications(PAGE_SIZE, page * PAGE_SIZE, notificationType, event_id);
-       if (!response) return;
-       setNotifications(response.notifications);
-       setTotal(response.total);
-    } catch(e) {
-      console.log(e)
+      const response = await getNotifications(
+        PAGE_SIZE,
+        page * PAGE_SIZE,
+        notificationType,
+        event_id
+      );
+      if (!response) return;
+      setNotifications(response.notifications);
+      setTotal(response.total);
+    } catch (e) {
+      console.log(e);
```

Comment:
We're not displaying any error messages to the user/admin would be great to have a `Toast` displaying async errors i.e `We couldn't get new notifications`